src/harness/values.yaml

```diff
@@ -184,6 +184,12 @@ sto:
 # -- overrides for Postgresql
 infra:
```

Comment:
dont you need to add global fields for values.yaml too?

Reply:
all do you need to edit the override-demo and override-prod files?

Reply:
> all do you need to edit the override-demo and override-prod files?

No, these changes go to values.yaml itself- to enable metrics for harness services. 
I'll make the changes.